BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.h

```diff
@@ -180,4 +170,13 @@
  **/ 
 - (void)setSharedLinkStorageDelegate:(id <BOXSharedLinkStorageProtocol>)delegate;
 
+/**
+ *  API base URLs.
+ **/
++ (NSString *)APIBaseURL;
```

Comment:
I was carrying over naming based on the property name. I could change it to end with 'URLString' instead, but probably unnecessary? Let me know if you prefer that.